File: testcases/paper/Function alias/Function.cpp
class Function {
    static int i, j;

    static int method() {
        return Function::i;
    }

    int (& aliasMethod)() = method;

    int method2() {
        return aliasMethod();
    }
};

// Function aliases don't count: LCOM4 = 2
// Functions aliases count:      LCOM4 = 1

File: testcases/paper/Function alias/Function.go
package Function

type Function struct {
    i,j int
}

func (c Function) method() {
    c.i = 1
}

func (c Function) method2() {
    c.j = 2
    var aliasMethod = c.method
    aliasMethod()
    return
}

// Function aliases don't count: LCOM4 = 2
// Functions aliases count:      LCOM4 = 1